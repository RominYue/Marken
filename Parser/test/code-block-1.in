    #include <cstdio>
    
    int main() {
        int a;
        scanf("%d", &a);
        return 0;
    }