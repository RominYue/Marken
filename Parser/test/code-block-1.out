<pre><code>#include &lt;cstdio&gt;

int main() {
    int a;
    scanf("%d", &amp;a);
    return 0;
}</code></pre>
